xla/mlir/backends/gpu/transforms/lmhlo_gpu_to_gpu_runtime.cc

```diff
@@ -839,6 +872,20 @@ class FusedAttentionBackwardLowering
     set_attr("bmm2_grad_gemm2_dot_dimension_numbers",
              op.getBmm2GradGemm2DotDimensionNumbers());
 
+    auto set_xi64 = [&](StringRef name, mlir::ArrayAttr array) {
```

Comment:
Up to you, but I think it would be slightly more readable if this function, and set_attr were moved into standalone functions.

Reply:
I don't want to move this into namespace `xla::gpu` because `convOp` have a same function name but with slightly different implementation and other op just directly inline this function multiple times. So this might cause confusion if moved to other places. I also don't want to make it a member function of the class `FusedAttentionBackwardLowering` because it is not used by any other member functions. So I think we should keep it this way.